qiskit/visualization/pulse/matplotlib.py

```diff
@@ -581,7 +581,7 @@ def _draw_table(self, figure, channels: Dict[Channel, EventsOutputChannels], dt:
                 # pylint: enable=unbalanced-tuple-unpacking
                 time, ch_name, data_str = data
                 # item
-                cell_value[r][3 * c + 0] = "t = %s" % time * dt
+                cell_value[r][3 * c + 0] = "t = %s" % time * dt  # TODO: worng operator priority?
```

Comment:
```suggestion
                cell_value[r][3 * c + 0] = "t = %s" % time * dt  # TODO: wrong operator priority?
```

Reply:
I think the intention was `t = %s" % (time * dt)`, otherwise multiplication is applied to string

```
time = 1
dt = 2
"t = %s" % time * dt
Out[14]: 't = 1t = 1'
```